lib/govuk_app_config/govuk_error.rb

```diff
@@ -11,9 +12,11 @@ def self.notify(exception_or_message, args = {})
     Raven.capture_exception(exception_or_message, args)
   end
 
+  def self.init(raven_configuration = DefaultConfiguration.new(Raven.configuration))
+    @raven_configuration = raven_configuration
+  end
+
   def self.configure
-    Raven.configure do |config|
-      yield(config)
-    end
+    yield @raven_configuration
```

Comment:
I did wonder if we needed the init method at all and could do:
```
Raven.configure do |raven_config|
  yield Configuration.new(raven_config)
end
```

Reply:
On second thoughts, it is better with an `.init` method since that'll preserve state in our delegator even if there are multiple `.configure` calls. So keep doing what you're doing here.